File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L5/08/Kasus4L5.ts
/**
 * Converts miles to kilometers.
 * @param miles The number of miles to convert.
 * @returns The number of kilometers.
 */
function convertMilesToKilo(miles: number): number {
    return miles * 1.609;
}

console.log("Miles\t\tKilometers");
console.log("-------------------------------");

// Use for loop
for (let i = 1; i <= 10; i++) {
    console.log(`${i}\t\t${convertMilesToKilo(i)}`);
}

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L5/08/Kasus4L5.kt
package IR

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus4L5 {
    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")
        println("-------------------------------")
        // Use for loop
        for (i in 1..10) {
            //Menampilkan hasil
            //i = miles
            println(i.toString() + "\t\t" + ConvertMilesToKilo(i))
        }
    }

    private fun ConvertMilesToKilo(miles: Int): Double {
        return miles * 1.609
    }
}
